src/lib/database/seed.ts

```diff
@@ -1,6 +1,13 @@
 import type { Database } from "./schema.ts"
 
-export const seed = async (db: Database) => {
+export type Seed = {
+  john_connected_account_id: "john_connected_account_id"
+  jane_connected_account_id: "jane_connected_account_id"
+  some_workspace_1: "some_workspace_1"
```

Comment:
I strongly dislike `some` as a prefix. Prefer no prefix or something useful about the workspace.

Reply:
really I always thought `some_` was a great way to mean `sample_` a la a code snippet

Reply:
maybe `seed_`? feels weird to not have a prefix considering the auto incrementing workspaces will be prefix-less